Assets/Scripts/Combat/Damageable.cs

```diff
@@ -393,7 +401,7 @@ private IEnumerator ElectricParalysis(BaseStateMachine baseStateMachine, int ind
         GAgent gAgent = GetComponent<GAgent>();
```

Comment:
Remove `GAgent gAgent = GetComponent<GAgent>();`  since there aren;t supposed to be any references to `GAgent `in `Damageable`